src/Umbraco.Web.UI.Client/src/views/dashboard/media/mediadashboardvideos.html

```diff
@@ -15,4 +15,4 @@ <h4>To get you started:</h4>
 	    	</div>
 	    </li>
 	</ul>
-</div>
\ No newline at end of file
+</div>

```

Comment:
Maybe just wrap the content inside a `<div>` like this:

```
<div ng-if="videos.length">
    <h4>To get you started:</h4>
    <ul class="thumbnails">
        <li class="span2" ng-repeat="video in videos">
            <div class="thumbnail" style="margin-right: 20px">
                <a target="_blank" href="{{video.link}}" title="{{video.title}}">
                    <img ng-src="{{video.thumbnail}}"  alt="{{video.title}}">
                </a>
            </div>
        </li>
    </ul>
</div>
```

Reply:
@bjarnef I considered it, but when I tested it, the empty `<ul>` didn't cause any issues so... I went without the extra DOM element.

Reply:
Well there shouldn't be empty `<ul>` elements without `<li>´ which is invalid HTML. It is probably not critical here because it is inside backoffice, but I think it a bit more clear with a wrapping `<div>` .  It could also be useful with a class on the wrapper if we need to style something specific on this "component"  :)

Reply:
@bjarnef it's quicker to fix it than to argue, so there it is 😄